### PR TITLE
Common container base image for runtime and build

### PR DESCRIPTION
**Pull request by joonas-fi**

### CODE REVIEW

**Review by maseabunikie — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,36 +1,24 @@
...
+
+# this:
+# 1) builds the application
+# 2) packages the application as .deb & writes it to /main_ws/
+RUN /packaging/build.sh
+
+#  ▲               runtime ──┐
+#  └── build                 ▼
+
+FROM ghcr.io/tiiuae/fog-ros-baseimage:sha-7857f7d
```
Comment (maseabunikie) [inline Dockerfile:13]: Should we use newer version of the image? -> 
https://github.com/tiiuae/fog-ros-baseimage/pkgs/container/fog-ros-baseimage/18451119?tag=sha-1cabd43

Comment (joonas-fi) [reply Dockerfile:13]: Good catch, yeah I think it'd be safest if all projects used the exact same version, and the version you pointed out is newer and has some important updates!

I think we should make soon a script that pulls `Dockerfile` from each project and print out which base image version it uses, and we should check that script before each version release

Comment (maseabunikie) [reply Dockerfile:13]: After our team brainstorming in slack. Should we update the dockerfiles with stable tag?

Comment (joonas-fi) [reply Dockerfile]: Yes. Done. Thanks again!

**Pull request merged**